scripts/ci/collect-coverage.sh

```diff
@@ -3,18 +3,19 @@
 # This script is meant to be executed on all "slave" machines that run coverage collection.
 
 COV_DIR=coverage
-COV_INFO="$COV_DIR/$JOB_NAME.info"
+COV_NAME=`echo $JOB_NAME | sed 's#/#-#g'`.info
+COV_INFO="$COV_DIR/$COV_NAME"
 
 # Build Mono and collect coverage on the test suite.
-CI_TAGS=collect-coverage,monolite scripts/ci/run-jenkins.sh
+CI_TAGS=collect-coverage,monolite,"$CI_TAGS" scripts/ci/run-jenkins.sh
```

Comment:
please quote the whole arg instead: `CI_TAGS="collect-coverage,monolite,$CI_TAGS"`

Reply:
OK, I'll change that, though these 2 forms should are absolutely identical.